docs/developer/running-saleor/debugging-emails.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Debugging emails
+sidebar_label: Debugging emails
+---
+
+[Saleor platform](https://github.com/mirumee/saleor-platform) use [Mailhog](https://github.com/mailhog/MailHog) for catching all the emails send by the Saleor. 
+
+![Assign product to collection](../screenshots/mailhog.png)
+
+Pros of using Mailhog during development:
```

Comment:
IMHO this chapter should convince user to configure UserEmailPlugin(or AdminEmailPlugin), instead of changeing env settings. Each of these actions can be done directly from the Dashboard's plugin configuration view. Additionally, you can upload there your own email templates, which will allow you to also debug email content (with described approach, we can only debug a communication Saleor -> Emailgatewat(mailhog)).